src/ray/common/ray_config_def.h

```diff
@@ -300,6 +300,9 @@ RAY_CONFIG(int64_t, task_rpc_inlined_bytes_limit, 10 * 1024 * 1024)
 /// pipelining task submission.
 RAY_CONFIG(uint32_t, max_tasks_in_flight_per_worker, 1)
 
+/// Maximum number of pending lease requests per scheduling category
+RAY_CONFIG(uint64_t, max_pending_lease_requests_per_scheduling_category, 1)
```

Comment:
Let's set this to 10 by default.

Reply:
Done

Reply:
I'd like to see this PR pass with it set to 10 though before we merge, even if it's off by default. Otherwise, we have code that is not covered by any nightly tests or performance tests.